README.md

```diff
@@ -301,6 +301,25 @@ In particular:
 
 
 
+#### Github using SSH
+To use terragrunt with SSH github authentication, use the following syntax. Note the `/` after 
+github.com instead of `:` and the uses of `//`. 
```

Comment:
I would change the heading title to "Using Terragrunt with private Git repos" and the first sentence to say: 

```
The easiest way to use Terragrunt with private Git repos is to use SSH authentication. Configure your Git account so you can use it with SSH (see the [guide for GitHub here](https://help.github.com/articles/connecting-to-github-with-ssh/) and use the SSH URL for your repo, prepended with `git::ssh://`:
```